cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -1248,11 +1241,6 @@ private void testTracingConfiguration(String type) {
 
         KafkaBridgeBuilder builder = new KafkaBridgeBuilder(this.resource);
         switch (type) {
-            case JaegerTracing.TYPE_JAEGER:
```

Comment:
Would it make sense to merge this method directly into `testOpenTelemetryTracingConfiguration`? I guess it is now used only form this one place, or?

Reply:
Yeah I was leaving there just in casa we will have a new tracing in the future ... but who knows :-)
Let me merge it.